### ISSUE TITLE
"Save as Google Doc" button on *everything*

### ISSUE THREAD

**Issue by schanzer** [bootstrapworld/curriculum#187]
From https://stackoverflow.com/questions/23815541/is-it-possible-to-upload-and-convert-an-rtf-file-to-a-gdoc-from-php-web-app and https://developers.google.com/drive/api/v2/reference/files/insert#examples, it appears that the drive SDK allows someone to upload an html file and have it auto-converted to a GDoc. 

We need a button, that we can affix to *any* web page we generate from `adoc`, that will do this. This is going to take some API reading and creativity (and possibly talking to Joe and Ben about the Drive API).

**Comment by schanzer** [comment]
Apparently it's easy to do a mediocre job:

In the <head> of every page: 
```
        <script>
          window.___gcfg = {
            parsetags: 'explicit'
          };
        </script>
        <script src="https://apis.google.com/js/platform.js" async defer></script>
```

In the body tag:
`<div id="body" onload="renderSaveToDrive();">`

After the Title:
```
 <div id="savetodrive-div"></div>
        <script>
		function renderSaveToDrive() {
            gapi.savetodrive.render('savetodrive-div', {
              src:"https://www.bootstrapworld.org/materials/spring2020/courses/datascience-remix/en-us/lessons/ds-correlations/",
              filename:"Bootstrap:Data Science - Correlations",
              sitename:"Bootstrap, Brown University"
            });
          }
        </script>
```

**Comment by schanzer** [comment]
Closed by f73b68c69b6b1e43e57dd82b7f91d3add4ce5711. Go, @ds26gte !

(I'll open a separate issue to track formatting improvements as we uncover them)
